### DISCUSSION TITLE
How to make a post request?

### DISCUSSION THREAD

**Discussion by hkargc** [php-webdriver/php-webdriver#911] [Q&A] [answered]
$driver->get('***');
The driver only gives the get method
How to make a post request?
Something like this:
$driver->post(url, {name:'', card:''});

**Comment by andrewnicols** [comment, marked as answer]
This driver is intended to control a browser by interacting as a human would. The `get()` function is the equivalent of a human typing in the address bar and visiting the URL. It is not designed to handle interactions like POST without doing something like a form submission, or interacting with the UI in such a way that client-side JS would submit a POST.

**Reply by hkargc** [reply]
Thank you for your reply
